core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -94,6 +94,11 @@ class BasicWorkflow extends EnsimeSpec
             case Some(OffsetSourcePosition(_, _)) =>
           }
 
+          val sourceFileLocation = symInfo.`type`.pos.get.asInstanceOf[OffsetSourcePosition].file
+          project ! SymbolDesignationsReq(Right(SourceFileInfo(sourceFileLocation, None, None)), -1, 9999, SourceSymbol.allSymbols)
+          val symbolDesignations = expectMsgType[SymbolDesignations]
+          symbolDesignations.syms should not be List.empty
```

Comment:
in scalatest you can just say `should not be ('empty)` and it works for any collection (check my syntax)